MarketingCodingAssignment/MarketingCodingAssignment/Views/Home/Index.cshtml

```diff
@@ -183,15 +183,15 @@
                 lines.push("<div class='panel-body'>");
                 if (item.tagline || item.runtime || item.revenue) {
                     lines.push("<div>");
-                        if (item.tagline) lines.push("<span class='tagline'><em>" + item.tagline + "</em></span>");
-                        if (item.tagline && (item.runtime || item.revenue)) lines.push(" | ");
-                        if (item.runtime) lines.push("<span> " + item.runtime + " minutes</span>");
-                        if (item.runtime && item.revenue) lines.push(" | ");
+                    if (item.tagline) lines.push("<span class='tagline'><em>" + item.tagline + "</em></span>");
+                    if (item.tagline && (item.runtime || item.revenue)) lines.push(" | ");
+                    if (item.runtime) lines.push("<span> " + item.runtime + " minutes</span>");
+                    if (item.runtime && item.revenue) lines.push(" | ");
                     if (item.revenue) lines.push("<span>" + item.revenue.toLocaleString("en-US", { style: "currency", currency: "USD" }) + "</span>");
                     lines.push("</div>");
                 }
 
-                if (item.releasedate) lines.push("<div>Initially released " + humanDate(item.releasedate) + "</div>");
+				if (item.releaseDate) lines.push("<div>Initially released " + new Date(item.releaseDate).toLocaleDateString() + "</div>");
```

Comment:
changing "releasedate" to "releaseDate" to match c# code. Also, providing a simple way to display a human readable date